pinot-query-runtime/src/test/java/org/apache/pinot/query/runtime/QueryRunnerTestBase.java

```diff
@@ -268,9 +274,18 @@ private static List<String> toH2FieldNamesAndTypes(org.apache.pinot.spi.data.Sch
         case DOUBLE:
           fieldType = "double";
           break;
+        case BOOLEAN:
+          fieldType = "BOOLEAN";
+          break;
         case BIG_DECIMAL:
           fieldType = "NUMERIC";
           break;
+        case BYTES:
+          fieldType = "BYTEA";
```

Comment:
typo? should this be `"BYTES"`?

Reply:
no, postgres and SQL standard calls this `BYTEA` (I think it stands for byte array)

Reply:
ah..I see. Thanks !!!